### ISSUE TITLE
Using arrays with fallback keys

### ISSUE THREAD

**Issue by vbichkovsky** [i18next/i18next#974]
Hello!

I have the following structure in my translations:

```
        "search": {
            "flagList": [
                ["basic", "Basic"],
                ["premium", "Premium"],
                ["public", "Public"],
                ["private", "Private"],
                ["active", "Active"],
                ["archived", "Archived"]
            ]
        }
```

and I'm using a common key fallback rule to look things up the hierarchy like this:

`t(['search.flagList', 'flagList'])`

The idea here is to reuse common things defined higher in hierarchy, and, when necessary, override them.

Simple `t('search.flagList')` and even `t(['search.flagList'])` work like expected, returning an array, while 

`t(['search.flagList', 'flagList'])`

returns 

`[ "flagList.0", "flagList.1", "flagList.2", "flagList.3", "flagList.4", "flagList.5" ]`

**Comment by jamuhl** [comment]
Rather sure has todo with https://github.com/i18next/i18next/blob/master/src/Translator.js#L242 will need to test later today and will provide a fix asap.

**Comment by jamuhl** [comment]
Could you try with i18next@10.0.2

**Comment by jamuhl** [comment]
was this solved with 10.0.2?

**Comment by jamuhl** [comment]
closing for now...should be solved in 10.0.2 but did not hear back from you....
